### ISSUE TITLE
CONST part of Expression produces error in setObjective

### ISSUE THREAD

**Issue by fserra** [SCIP-Interfaces/PySCIPOpt#28]
the code in set objective says
```python
assert isinstance(coeffs, Expr)
for term, coef in coeffs.terms.items():
    var = <Variable>term[0]
    PY_SCIP_CALL(SCIPchgVarObj(self._scip, var.var, coef))
```
which produces an error when `term` is `()`, ie, the `CONST` part of `coeffs`.

A solution is to put an `if coef != 0.0:` before. It this okay?

**Comment by fserra** [comment]
it seems that a commit msg like "fixes #28" doesn't close this.
Fix with 8198f78

**Comment by leethargo** [comment]
The fix is incomplete, unfortunately. The call to `setObjective` still fails on nonlinear expressions, or with non-zero offsets. I'll prepare a patch.
